assets/app/scripts/services/cachedTemplate.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .service("CachedTemplateService", function(){
+
+    var template = {};
```

Comment:
I'd rather set this to null than the empty object, then callers can just check `if (template) { ... } `instead of `if (_.isEmpty(template)) { ... }`